testing/src/test/java/io/stargate/it/driver/CqlSessionHelper.java

```diff
@@ -0,0 +1,22 @@
+package io.stargate.it.driver;
+
+import com.datastax.oss.driver.api.core.CqlSession;
+
+/**
+ * Utility methods to work with sessions in Stargate tests. To use this object, declare it as a
+ * parameter of a test method and {@link CqlSessionExtension} will inject it.
+ */
+public interface CqlSessionHelper {
```

Comment:
Keeping the name generic, in case we find other stuff to add here in the future.